spanner/google/cloud/spanner_v1/client.py

```diff
@@ -95,9 +95,17 @@ class Client(ClientWithProject):
                         client. If not provided, defaults to the Google
                         Application Default Credentials.
 
+    :type client_info: :class:`google.api_core.client_info.ClientInfo`
+    :param client_info:
+        (Optional) The client info used to send a user-agent string along with
+        API requests. If ``None``, then default info will be used. Generally,
+        you only need to set this if you're developing your own library or
+        partner tool.
+
     :type user_agent: str
-    :param user_agent: (Optional) The user agent to be used with API request.
-                       Defaults to :const:`DEFAULT_USER_AGENT`.
+    :param user_agent:
+        (Deprecated) The user agent to be used with API request.
+        Not used.  Defaults to :const:`DEFAULT_USER_AGENT`.
```

Comment:
Default value doesn't really matter since it's not used, right? I recommend removing that sentence from the docs (and maybe change the default to `None`).

Reply:
Sure.  In fact, I've gone further and added an actual deprecation warning for passing any non-None value in 7e71bf6.